src_assets/common/assets/web/configs/tabs/Inputs.vue

```diff
@@ -138,56 +125,49 @@ const config = ref(props.config)
     </div>
 
     <!-- Always send scancodes -->
-    <div class="mb-3" v-if="config.keyboard === 'enabled' && platform === 'windows'">
-      <label for="always_send_scancodes" class="form-label">{{ $t('config.always_send_scancodes') }}</label>
-      <select id="always_send_scancodes" class="form-select" v-model="config.always_send_scancodes">
-        <option value="disabled">{{ $t('_common.disabled') }}</option>
-        <option value="enabled">{{ $t('_common.enabled_def') }}</option>
-      </select>
-      <div class="form-text">{{ $t('config.always_send_scancodes_desc') }}</div>
-    </div>
+    <Checkbox v-if="config.keyboard === 'enabled' && platform === 'windows'"
+              class="mb-3"
+              id="always_send_scancodes"
+              locale-prefix="config"
+              v-model="config.always_send_scancodes"
+              default="true"
+    ></Checkbox>
 
     <!-- Mapping Key AltRight to Key Windows -->
-    <div class="mb-3" v-if="config.keyboard === 'enabled'">
-      <label for="key_rightalt_to_key_win" class="form-label">{{ $t('config.key_rightalt_to_key_win') }}</label>
-      <select id="key_rightalt_to_key_win" class="form-select" v-model="config.key_rightalt_to_key_win">
-        <option value="disabled">{{ $t('_common.disabled') }}</option>
-        <option value="enabled">{{ $t('_common.enabled_def') }}</option>
-      </select>
-      <div class="form-text">{{ $t('config.key_rightalt_to_key_win_desc') }}</div>
-    </div>
+    <Checkbox v-if="config.keyboard === 'enabled'"
+              class="mb-3"
+              id="key_rightalt_to_key_win"
+              locale-prefix="config"
+              v-model="config.key_rightalt_to_key_win"
+              default="false"
```

Comment:
```suggestion
              default="true"
```

I think this one is reversed?

Reply:
Actually no, according to source code it was always meant to be false by default:
![image](https://github.com/user-attachments/assets/9df70345-cef5-42f3-9ba1-a1c988f6537a)